### ISSUE TITLE
Inform of Error when Reentry Import Fails with Status other than 200

### ISSUE THREAD

**Issue by ysteven13** [RimSort/RimSort#678]
### Release Type

Compiled (Windows x86-64)

### Version

v1.0.10

### Operating System

Windows 11

### Describe the bug

**Issue:**

Reentry Import Fails silently if request to reentry returns status code != 200. 

As long as an exception is not thrown during the request, the code will return without logging any error after [here](https://github.com/RimSort/RimSort/blob/1a380b5ad026d8cf7c74eebd82727403901ff267/app/utils/rentry/wrapper.py#L164-L176). 

As such `self.package_ids` is still empty when `import_rentry_link()` returns
and `_do_import_list_rentry()` returns early [here](https://github.com/RimSort/RimSort/blob/1a380b5ad026d8cf7c74eebd82727403901ff267/app/views/main_content_panel.py#L1329C8-L1329C42).

**Expected Behavior:**

If reentry does not return 200 then either log an error or open a popup informing the user what the error response was. 

### Code of Conduct

I agree to follow this project's Code of Conduct

### Duplicate Issue Check

I've checked for similar issues and didn't find anything

### Wiki/FAQ Check

I've checked the Wiki for a solution and didn't find a solution
